crypto/rand/rand_crng_test.c

```diff
@@ -30,7 +30,7 @@ int rand_crngt_get_entropy_cb(unsigned char *buf)
     while ((n = rand_pool_acquire_entropy(crngt_pool)) != 0)
```

Comment:
This doesn't make any sense, calling the function again will normally not acquire more entropy.

Reply:
The crngt_pool also seems to be unused, this should be an infinite loop.

Reply:
Calling `rand_pool_acquire_entroy` again should get another match of entropy.  I should put a guard counter around this construction to prevent an infinite loop is the sources cannot return enough entropy.

`crngt_pool` the entropy it contains is extracted, digested and zeroed.  NIST mandates than the first block of entropy not be used for anything other than this test.